feature

asadmisrani opened the pull request:
All files are created

HTML + basic JavaScript Capstone

Loom video link: https://www.loom.com/share/356aee85b45240a6b7cb4632127f95bb

personalize the content of the page in this project.
Colors.
Typography: font face, size, and weight as a free font similar to the one used in the original design.
Layout: composition and space between elements.
The pages look almost identical to the original design. Minor adjustments like text or image changes are acceptable.

I build only these 2 pages:
The Home page.
The About page.

Each of these pages has versions for 2 different screen sizes:

Mobile: up to 768px wide.
Desktop: 768px or wider.

Interactions:
Links:
The home page has a link in the menu to the About page.
The logo in the header links to the home page.

Mobile menu
When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
There are no guidelines for the mobile menu in the docs, but I implement it so it is consistent with the design (colors, typography, spacing, etc.).
The mobile menu has a close (X) button that closes the menu.

Dynamic page:
The section "Featured speakers" is created dynamically in JavaScript.
I have used a JavaScript variable with the data about the speakers and used it when the page loads to create the HTML for this section dynamically.

asadmisrani committed:
All files are created

asadmisrani committed:
README.md file and Index.html file were modified

asadmisrani committed:
Index.html file were modified

asadmisrani committed:
Index.html file and Style.css file were modified

asadmisrani committed:
Index.html file were modified

asadmisrani committed:
Style.css file were modified

asadmisrani committed:
Main.js file were modified

asadmisrani committed:
.eslintrc.json file were modified

asadmisrani committed:
About.html file created

asadmisrani committed:
Style.css file and linters file were modified

asadmisrani committed:
Linters file were modified

asadmisrani committed:
Linters.yml file were modified

asadmisrani committed:
Linters file were modified

asadmisrani committed:
Linters file were modified

asadmisrani committed:
Linters file modified

asadmisrani committed:
Linters file were modified

asadmisrani committed:
Linters file were modified

asadmisrani committed:
About.js file and Style.css file were modified

asadmisrani committed:
Index.html file were modified

asadmisrani committed:
Linters.yml file were modified

asadmisrani committed:
Linters file were modified

Zuhaib042 reviewed (commented):
Hi @asadmisrani 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a readme in which the sub-sections under `Getting Started` are not clear to the visitor, how he should set up the project and how and run it locally. and you can also remove `Run tests` as your project doesn't include testing.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

asadmisrani committed:
README.md file were modified

asadmisrani committed:
README.md file were modified

binhussen commented:
hi @asadmisrani, would you remove `Run tests` sections as previous reviewer suggested you, to proceed your review

asadmisrani commented:
Which run tests section? would I need to remove it? It is not a project
requirement.

On Fri, Apr 21, 2023 at 12:47 AM Mohammed Hussen ***@***.***>
wrote:

> hi @asadmisrani <https://github.com/asadmisrani>, would you remove Run
> tests sections as previous reviewer suggested you, to proceed your review
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/asadmisrani/capstonemodule1.github.io/pull/1#issuecomment-1516860474>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYQGAKHOUZ3SCS7PZDJ2YUTXCGHDXANCNFSM6AAAAAAXCIIF5M>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

asadmisrani committed:
README.md file were modified

binhussen commented:
yes you have to remove it, as you haven’t the
 test

asadmisrani commented:
I removed the run tests section in the description. Now please check my
project.

On Fri, Apr 21, 2023 at 12:53 AM asad shareef ***@***.***>
wrote:

> Which run tests section? would I need to remove it? It is not a project
> requirement.
>
> On Fri, Apr 21, 2023 at 12:47 AM Mohammed Hussen ***@***.***>
> wrote:
>
>> hi @asadmisrani <https://github.com/asadmisrani>, would you remove Run
>> tests sections as previous reviewer suggested you, to proceed your review
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/asadmisrani/capstonemodule1.github.io/pull/1#issuecomment-1516860474>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AYQGAKHOUZ3SCS7PZDJ2YUTXCGHDXANCNFSM6AAAAAAXCIIF5M>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

asadmisrani commented:
Yes I removed it.

On Fri, Apr 21, 2023 at 12:55 AM asad shareef ***@***.***>
wrote:

> I removed the run tests section in the description. Now please check my
> project.
>
> On Fri, Apr 21, 2023 at 12:53 AM asad shareef ***@***.***>
> wrote:
>
>> Which run tests section? would I need to remove it? It is not a project
>> requirement.
>>
>> On Fri, Apr 21, 2023 at 12:47 AM Mohammed Hussen <
>> ***@***.***> wrote:
>>
>>> hi @asadmisrani <https://github.com/asadmisrani>, would you remove Run
>>> tests sections as previous reviewer suggested you, to proceed your
>>> review
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/asadmisrani/capstonemodule1.github.io/pull/1#issuecomment-1516860474>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AYQGAKHOUZ3SCS7PZDJ2YUTXCGHDXANCNFSM6AAAAAAXCIIF5M>
>>> .
>>> You are receiving this because you were mentioned.Message ID:
>>> ***@***.***>
>>>
>>

binhussen commented on README.md:
```diff
@@ -1 +1,222 @@
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to Microverse.
+
```
- Kindly, as you are using the [Cindy Shin](https://www.behance.net/adagio07) design you have to give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 👍

binhussen commented on index.html:
```diff
@@ -0,0 +1,245 @@
+  <body>
+    <header class="header">
+      <nav>
+        <div class="navbar navbar-expand-lg navbar-dark bg-dark navbar-social">
+          <div class="container-fluid ">
+            <div class="collapse navbar-collapse justify-content-end
+              navbar-nav-social" id="navbarNav">
+              <ul class="navbar-nav nav-bar-items-social">
+                <li class="nav-item">
+                  <a class="nav-link active nav-icon" aria-current="page"
+                    href="#"><i class="bi bi-facebook"></i></a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link active" href="#"><i class="bi bi-twitter"></i></a>
+                </li>
```
- Kindly, the social icons and language sections are not part of the  navigation so please make them outside  the `nav` tag for semantic reasons 👍
- Kindly, you have given unnecessary space between the social icons and language please update it to have a better design and follow the given design 👍
### your design 👇
![2023-04-20_23-06](https://user-images.githubusercontent.com/52010789/233476721-5b7881c0-10ec-4db3-9f56-e591ae49b89f.png)
### Given design 👇
![2023-04-20_23-09](https://user-images.githubusercontent.com/52010789/233476913-52c198e1-e625-40a9-a7f0-13edca00d79f.png)

binhussen commented on index.html:
```diff
@@ -0,0 +1,245 @@
+        </div>
+      </section>
+
+      <section class="speakers" id="speakers">
+        <div class="featured">
+          <h4>Featured Speakers</h4>
+        </div>
+        <div class="main-bar"></div>
+
+        <div class="grid-container">
+
```
- Kindly add the for `hover` state white border to improve the user experience 👍
![2023-04-20_23-11](https://user-images.githubusercontent.com/52010789/233477630-8f2ad8bc-d4e1-430d-8615-1a5b832e2e3e.png)

Munsa1 replied:
- [ ] Again here as requested by the previous reviewer, kindly add a hover effect of a white border around the articles to improve user experience. I think the previous reviewer meant a `hover effect` as is shown in the given design

Given design
![image](https://user-images.githubusercontent.com/81932478/233687967-fb701f23-e991-49bd-afe9-6fd78e015e82.png)

#### Note that the fourth article in the screenshot above has a white border showing that the cursor is hovering over it. 

Your design
![image](https://user-images.githubusercontent.com/81932478/233688330-500168e2-ced3-4168-8fc6-0b0db84718dd.png)

binhussen commented on index.html:
```diff
@@ -0,0 +1,245 @@
+        </div>
+        <div class="main-bar"></div>
+        <div class="main-activities">
+          <table class="table table-borderless t-main">
+            <tbody>
+              <tr class="main-row">
+                <td class="t-img"><img class="main-img" src="Images/main_01.png"
+                    alt="Speaker Icon"></td>
+                <td class="t-title">Lecture</td>
+                <td class="t-description">Listen to the speakers from various
+                  countries about
+                  they cientific studies related to evolution.</td>
+              </tr>
+            </tbody>
+          </table>
```
- Kindly, these main program lists are articles so make them inside `article` for semantic reasons 👍

binhussen commented on main.js:
```diff
@@ -0,0 +1,129 @@
+  }
+  // card.classList.add('card', postData.class);
+  card.innerHTML = `
+    <div class="featuredSpeakers">
+      <img class="cardImg" src="${postData.portrait}" alt="Speaker Image">
+      <div class="speakersInfo">
+        <h5 class="speakerName">${postData.name}</h5>
+        <h6 class="speakerPosition">${postData.position}</h6>
+        <div class="speakerBar"></div>
+        <p class="speakerExperience">${postData.resume}</p>
+      </div>
+    </div>
```
- Great job for making the speaker lists dynamic, I think it would be better if you used `article` for wrapping each speaker for semantic reasons 👍

binhussen commented on README.md:
```diff
@@ -1 +1,222 @@
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://asadmisrani.github.io/capstonemodule1.github.io/)
+
+- This is the [Loom Walktrough Video](https://www.loom.com/share/60bcd67bfb264ba081ae8ac442537624) of my project.
```
- Kindly your sound is not audible and The video doesn’t contain a highlight about a piece of the code that you find interesting or is proud of making it. Please record another video as it is part of the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md) 👍

Munsa1 replied:
- [ ] As requested by the previous reviewer kindly ensure that the audio from your video recording is clear and audible and that the reviewer is able to hear you speak, highlighting a piece of code you found interesting or are proud of making.

binhussen reviewed (changes requested):
Hi @asadmisrani , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<img src="https://media0.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

asadmisrani committed:
README.md, About.html, Index.html, main.js and Style.css files are modification

asadmisrani committed:
Index.html and About.html file were modified

asadmisrani committed:
Style.css file were modified

asadmisrani committed:
README.md file were modified

Munsa1 commented on index.html:
```diff
@@ -0,0 +1,242 @@
+          <div class="collapse navbar-collapse justify-content-end
+            navbar-nav-social" id="navbarNav"><div>
+              <ul class="navbar-nav nav-bar-items-social">
+
+                <li><a href=""><img src="images/facebook.png" alt="facebook
+                      logo"></a></li></div>
+              <li><a href=""><img src="images/twitter.png" alt="twitter logo"></a></li>
```
- [ ] Great job working on the social media icons, however, they are not displaying due to broken links. This is because your links start with a small letter `i` for the images folder when your actual folder has the capital letter `I`. Kindly fix this

Your design
![image](https://user-images.githubusercontent.com/81932478/233684259-c19c4ad6-9dab-4729-9d10-4baa6ea2f96c.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/233684463-6406b0ce-3443-40fe-8abf-05686e84f72e.png)

Munsa1 reviewed (changes requested):
Hi @asadmisrani, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

asadmisrani committed:
Index.html file and Style.css file were modified

asadmisrani committed:
README.md file were modified

asadmisrani commented:
Thanks for the feedback!

All issues are solved please now check again.

On Fri, Apr 21, 2023 at 9:35 PM Munsa Mibenge ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @asadmisrani <https://github.com/asadmisrani>,
>
> Great work on making the changes 👏
> You've done well implementing some of the requested changes, but there are
> still some which aren't addressed yet.
> Suggested changes
>
> *Check the comments under the review.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> In index.html
> <https://github.com/asadmisrani/capstonemodule1.github.io/pull/1#discussion_r1173949655>
> :
>
> > +
> +                <li><a href=""><img src="images/facebook.png" alt="facebook
> +                      logo"></a></li></div>
> +              <li><a href=""><img src="images/twitter.png" alt="twitter logo"></a></li>
>
>
>    - Great job working on the social media icons, however, they are not
>    displaying due to broken links. This is because your links start with a
>    small letter i for the images folder when your actual folder has the
>    capital letter I. Kindly fix this
>
> Your design
> [image: image]
> <https://user-images.githubusercontent.com/81932478/233684259-c19c4ad6-9dab-4729-9d10-4baa6ea2f96c.png>
>
> Given design
> [image: image]
> <https://user-images.githubusercontent.com/81932478/233684463-6406b0ce-3443-40fe-8abf-05686e84f72e.png>
> ------------------------------
>
> In README.md
> <https://github.com/asadmisrani/capstonemodule1.github.io/pull/1#discussion_r1173961700>
> :
>
> > +
> +- This is the [Loom Walktrough Video](https://www.loom.com/share/60bcd67bfb264ba081ae8ac442537624) of my project.
>
>
>    - As requested by the previous reviewer kindly ensure that the audio
>    from your video recording is clear and audible and that the reviewer is
>    able to hear you speak, highlighting a piece of code you found interesting
>    or are proud of making.
>
> ------------------------------
>
> In index.html
> <https://github.com/asadmisrani/capstonemodule1.github.io/pull/1#discussion_r1173968143>
> :
>
> > +      <section class="speakers" id="speakers">
> +        <div class="featured">
> +          <h4>Featured Speakers</h4>
> +        </div>
> +        <div class="main-bar"></div>
> +
> +        <div class="grid-container">
> +
>
>
>    - Again here as requested by the previous reviewer, kindly add a hover
>    effect of a white border around the articles to improve user experience. I
>    think the previous reviewer meant a hover effect as is shown in the
>    given design
>
> Given design
> [image: image]
> <https://user-images.githubusercontent.com/81932478/233687967-fb701f23-e991-49bd-afe9-6fd78e015e82.png>
> Note that the fourth article in the screenshot above has a white border
> showing that the cursor is hovering over it.
>
> Your design
> [image: image]
> <https://user-images.githubusercontent.com/81932478/233688330-500168e2-ced3-4168-8fc6-0b0db84718dd.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/asadmisrani/capstonemodule1.github.io/pull/1#pullrequestreview-1396047577>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYQGAKADHB3UKEGFBGQ35YLXCKZMPANCNFSM6AAAAAAXCIIF5M>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

Shisui6 reviewed (approved):
Hi @asadmisrani, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Home and about pages created ✅
- Project personalized ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

asadmisrani commented:
Acknowledged.

On Sat, 22 Apr 2023, 01:07 Okemdi Udeh, ***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> Hi @asadmisrani <https://github.com/asadmisrani>,
> Approved 🪄
>
> Your project is complete! There is nothing else to say other than... it's
> time to merge it 💎
> Congratulations! 🎉
> Highlights
>
>    - Home and about pages created ✅
>    - Project personalized ✅
>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix won't stop the approval of this
> PR. However, I strongly recommend you consider them as they can improve
> your code. Some of them were simply missed by the previous reviewer, and
> addressing them will really improve your application.*
>
> You can also consider the following:
>
> N/A
>
> Cheers and Happy coding!🍻💻
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561>, you have
> a limited number of reviews per project (check the exact number in your
> Dashboard). If you think the code review was unfair, you can request a
> second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/asadmisrani/capstonemodule1.github.io/pull/1#pullrequestreview-1396341438>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AYQGAKFUW4VEL2JFPVD7FY3XCLSHDANCNFSM6AAAAAAXCIIF5M>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>
